python/python/ert/enkf/data/gen_data.py

```diff
@@ -30,6 +30,18 @@ def getData(self):
         GenData.cNamespace().export_data( self , data )
         return data
 
+    def __getitem__( self, idx ):
+        ls = len(self)
+        if isinstance( idx, int ):
+            if idx < 0:
+                idx += ls
+            if 0 <= idx < ls:
+                return self.getData()[idx]
```

Comment:
The `GenData` instances can potentially be *large* - they are typically used for seismic data. That means that creating the  full data vector and then only extracting one element from that vector looks very costly. Should rather use `gen_data_iget_double( )` to get one scalar value.